### PR TITLE
Core 2.2.11

### PR DESCRIPTION
**Pull request by VascoPi**
PURPOSE
Update RadeonProRenderSDK to core_2.2.11.

EFFECT OF CHANGE
Updated RadeonProRenderSDK to core_2.2.11.

TECHNICAL STEPS
Updated RadeonProRenderSDK submodule.

### CODE REVIEW

**Review by bnagirniak — APPROVED** [review]
Build on CIS is ok, tests are also ok. Waiting for approval from QA.

**Review by DagerD — APPROVED** [review]

**Comment by nlarionov** [comment]
After discussion with Atsushi, we can approve this PR. Core team know about issue with performance: https://amdrender.atlassian.net/browse/RPR-994

**Review by nlarionov — APPROVED** [review]

**Pull request merged**
